### PR TITLE
Setup New Base commit - f2134db

### PR DESCRIPTION
**Pull request by adambkaplan**
Set up a new branch as our default, based on the latest upstream commit.

This includes the following carry patches:

- Configuring HACBS
- Proto definitions for log forwarding/storage
- Initial implementation of log forwarding/storage

### CODE REVIEW

File: .tekton/tekton-results-api-pull-request.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+# Triggers Tekton Results api image build on pull request
+apiVersion: tekton.dev/v1beta1
+kind: PipelineRun
+metadata:
+  name: tekton-results-api-on-pull-request
+  annotations:
+    pipelinesascode.tekton.dev/on-event: "[pull_request]"
+    # Use pipelines-svc-0.1-6e05d4d branch since we are using openshift-pipeline/tektoncd-results and not upstream
+    pipelinesascode.tekton.dev/on-target-branch: "[pipelines-svc-0.1-6e05d4d]"
```
Comment (avinal) [inline .tekton/tekton-results-api-pull-request.yaml]:
```suggestion
    pipelinesascode.tekton.dev/on-target-branch: "[pipeline-svc-0.1-f2134db]"
```
Update branch ref

File: .tekton/tekton-results-api-push.yaml [hunk 2]
```diff
@@ -0,0 +1,28 @@
+---
+# Triggers Tekton Results api image build on push
+apiVersion: tekton.dev/v1beta1
+kind: PipelineRun
+metadata:
+  name: tekton-results-api-on-push
+  annotations:
+    pipelinesascode.tekton.dev/on-event: "[push]"
+    # Use pipelines-svc-0.1-6e05d4d branch since we are using openshift-pipeline/tektoncd-results and not upstream
+    pipelinesascode.tekton.dev/on-target-branch: "[pipelines-svc-0.1-6e05d4d]"
```
Comment (avinal) [inline .tekton/tekton-results-api-push.yaml]:
```suggestion
    pipelinesascode.tekton.dev/on-target-branch: "[pipeline-svc-0.1-f2134db]"
```

File: .tekton/tekton-results-watcher-pull-request.yaml [hunk 3]
```diff
@@ -0,0 +1,28 @@
+---
+# Triggers Tekton Results watcher image build on pull request
+apiVersion: tekton.dev/v1beta1
+kind: PipelineRun
+metadata:
+  name: tekton-results-watcher-on-pull-request
+  annotations:
+    pipelinesascode.tekton.dev/on-event: "[pull_request]"
+    # Use pipelines-svc-0.1-6e05d4d branch since we are using openshift-pipeline/tektoncd-results and not upstream
+    pipelinesascode.tekton.dev/on-target-branch: "[pipelines-svc-0.1-6e05d4d]"
```
Comment (avinal) [inline .tekton/tekton-results-watcher-pull-request.yaml]:
```suggestion
    pipelinesascode.tekton.dev/on-target-branch: "[pipeline-svc-0.1-f2134db]"
```

File: .tekton/tekton-results-watcher-push.yaml [hunk 4]
```diff
@@ -0,0 +1,28 @@
+---
+# Triggers Tekton Results watcher image build on push
+apiVersion: tekton.dev/v1beta1
+kind: PipelineRun
+metadata:
+  name: tekton-results-watcher-on-push
+  annotations:
+    pipelinesascode.tekton.dev/on-event: "[push]"
+    # Use pipelines-svc-0.1-6e05d4d branch since we are using openshift-pipeline/tektoncd-results and not upstream
+    pipelinesascode.tekton.dev/on-target-branch: "[pipelines-svc-0.1-6e05d4d]"
```
Comment (avinal) [inline .tekton/tekton-results-watcher-push.yaml]:
```suggestion
    pipelinesascode.tekton.dev/on-target-branch: "[pipeline-svc-0.1-f2134db]"
```

File: images/api/Dockerfile [hunk 5]
```diff
@@ -0,0 +1,18 @@
+# Docker configuration to build Tekton Results watcher image
+FROM registry.access.redhat.com/ubi9/go-toolset:1.17 as builder
```
Comment (avinal) [inline images/api/Dockerfile]: We must upgrade to go 1.18 or our CI is almost useless. In PR #10 you have added 
https://github.com/openshift-pipelines/tektoncd-results/pull/10/files#diff-e3e3626323997573b9caec04c0895ba573f528b38f590d8f1fafd02c2b4bc5afR2 this image. We will have to see if this works now

**Review by avinal — CHANGES_REQUESTED** [review]

File: .tekton/tekton-results-api-pull-request.yaml (hunk 1 above)
Comment (adambkaplan) [reply .tekton/tekton-results-api-pull-request.yaml]: done!

File: images/api/Dockerfile (hunk 5 above)
Comment (adambkaplan) [reply images/api/Dockerfile]: I've upgraded to use the 1.18 image. I also missed the copying of the vendor directory. Upstream results now uses vendored code, so we should not have `go mod download` instructions.

File: images/api/Dockerfile [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+
+COPY go.mod go.mod
+COPY go.sum go.sum
+COPY vendor/ vendor/
+COPY cmd/ cmd/
+COPY proto/ proto/
+COPY pkg/ pkg/
+
+RUN go mod download
+
```
Comment (avinal) [inline images/api/Dockerfile]:
```suggestion
```
Remove this line in favor of vendor

File: images/watcher/Dockerfile [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+
+COPY go.mod go.mod
+COPY go.sum go.sum
+COPY vendor/ vendor/
+COPY cmd/ cmd/
+COPY proto/ proto/
+COPY pkg/ pkg/
+
+RUN go mod download
+
```
Comment (avinal) [inline images/watcher/Dockerfile]:
```suggestion
```

File: images/api/Dockerfile (hunk 5 above)
Comment (avinal) [reply images/api/Dockerfile]: Yes, You forgot to remove the lines IG.

**Comment by adambkaplan** [comment]
Merging this PR in the interim - our CI cluster is down for a bit.

**Pull request merged**